main.go

```diff
@@ -557,21 +605,60 @@ corectl eval by "Region" // Returns the values for dimension "Region"`,
 
 	removeCmd = &cobra.Command{
 		Use:   "remove",
-		Short: "Remove one or mores generic entities (dimensions, measures, objects) in the app",
-		Long:  "Remove one or mores generic entities (dimensions, measures, objects) in the app",
+		Short: "Remove entities (connections, dimensions, measures, objects) in the app or the app itself",
+		Long:  "Remove one or mores generic entities (connections, dimensions, measures, objects) in the app",
 
 		PersistentPreRun: func(ccmd *cobra.Command, args []string) {
 			corectlCommand.PersistentPreRun(corectlCommand, args)
-			viper.BindPFlag("app", ccmd.PersistentFlags().Lookup("app"))
 			viper.BindPFlag("engine", ccmd.PersistentFlags().Lookup("engine"))
 			viper.BindPFlag("ttl", ccmd.PersistentFlags().Lookup("ttl"))
-			viper.BindPFlag("no-save", ccmd.PersistentFlags().Lookup("no-save"))
```

Comment:
Why are we removing these?

Reply:
Since the `corectl remove app` command does not have it.
(you would run `corectl remove app <app-id>` not `corectl remove app -app=<app-id>`